docs/env_vars.md

```diff
@@ -10,11 +10,11 @@ host name, to ignore the resolve proxy.
 **`HTTPS_PROXY`** - If the https proxy environment variable exists and the url
 protocol is https, use this proxy.
 
-**`HTTP_PROXY`** - If the http proxy environmet variable exists and the url
+**`HTTP_PROXY`** - If the http proxy environment variable exists and the url
 protocol is http or https, use this proxy.
 
 **`GITHUB_TOKEN`** - A GitHub personal access token with no additional scopes
 required. When created, it will just have public-access. This can be set at
 [https://github.com/settings/tokens][1].
 
-[1]: https://github.com/settings/tokens
\ No newline at end of file
+[1]: https://github.com/settings/tokens
```

Comment:
Why is this last line changed? Could you only change the part of the markdown that actually changed? Thanks!